### DISCUSSION TITLE
Plot color

### DISCUSSION THREAD

**Discussion by laakkti** [opensource9ja/danfojs#445] [Q&A]
Hi,

I can't figure out how to define the plotting color (the default seem to be like blue).

The other color is possible define like this. etc. plot_color, color, plot_fgcolor not working.
const layout = {
      width: 1000,
      plot_bgcolor: "#133863",
      paper_bgcolor: "#133863",            
      yaxis: {
        title: "km",
        color:"#00FF00"
      },
      xaxis: {
        title: "Month",
        color:"#00FFFF"        
      },      
    };

Br. Timo

**Comment by risenW** [comment]
Hello @laakkti You can't do that at the moment. We have to add support for it
